totalRP3/tools/Locale.lua

```diff
@@ -1062,6 +1062,16 @@ Putting your companion name in [brackets] will allow color and icon customizatio
 
 Importing content coming from a different version may cause issues in case of incompatibilities. Do you want to proceed anyway?]],
 
+	--*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
```

Comment:
New locales should now be placed after the
```
------------------------------------------------------------------------------------------------
--- PLACE LOCALIZATION NOT ALREADY UPLOADED TO CURSEFORGE HERE
--- THEN MOVE IT UP ONCE IMPORTED
------------------------------------------------------------------------------------------------
```
message at [the end of the locale](https://github.com/Ellypse/Total-RP-3/blob/master/totalRP3/tools/Locale.lua#L1255), so I know what needs to be uploaded to Curse.